libs/template/presets/storefront/experience/header.ts

```diff
@@ -105,7 +107,24 @@ export const HeaderTemplate: ExperienceComponent = {
                 icon: IconTypes.User,
                 rules: [{ hideByRule: 'USER.!AUTHENTICATED' }],
               },
-              components: [{ type: 'oryx-auth-login-link' }],
+              components: [
+                featureVersion >= '1.1'
+                  ? {
+                      type: 'oryx-content-link',
+                      options: {
+                        icon: IconTypes.User,
+                        url: '/account/overview',
+                        appearance: ContentLinkAppearance.Dropdown,
+                      },
+                      content: {
+                        data: {
+                          text: i18n('user.account'),
+                        },
+                      },
+                    }
+                  : {},
```

Comment:
Can we avoid adding an empty object here? I'd rather keep the component list as-is and only add a component when the featureVersion is used.